package.json

```diff
@@ -14,7 +14,8 @@
     "@oclif/plugin-commands": "2.2",
     "@oclif/plugin-help": "5.2",
     "@oclif/plugin-not-found": "2.3",
-    "ketting": "8.0.0-alpha.2"
+    "json5": "^2.2.3",
```

Comment:
I'm off hats now, I'd rather see pinned dependencies please. I get why hats make sense for libraries, but this isn't a library.